src/Components/Contact/ContactForm.js

```diff
@@ -23,7 +24,13 @@ function ContactForm() {
             setError("El email debe tener un formato valido")
             return
         }
-        console.log(contact)
+        try{
+            await post('http://localhost:3001/contacts',contact)
```

Comment:
Se podria hacer en algun servicio en vez de hacerlo directo con axios en el contactForm

Reply:
> Buenas Pablo, es un servicio el post, como decis de hacerlo? El vie., 9 de septiembre de 2022 07:48, GauntStrelok < ***@***.***> escribió:
> […](#)
> ***@***.**** commented on this pull request. ------------------------------ In src/Components/Contact/ContactForm.js <[#28 (comment)](https://github.com/alkemyTech/OT263-client/pull/28#discussion_r966913144)> : > @@ -23,7 +24,13 @@ function ContactForm() { setError("El email debe tener un formato valido") return } - console.log(contact) + try{ + await post('http://localhost:3001/contacts',contact) Se podria hacer en algun servicio en vez de hacerlo directo con axios en el contactForm — Reply to this email directly, view it on GitHub <[#28 (review)](https://github.com/alkemyTech/OT263-client/pull/28#pullrequestreview-1102115796)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ARJSBLOSMETA67MIALIEHNTV5MIWFANCNFSM6AAAAAAQHYXBNA> . You are receiving this because you authored the thread.Message ID: ***@***.***>

Ah, tenes razon, lo vi mal, hace pull de main para arreglar conflictos entonces